### ISSUE TITLE
Black screen (so strange)

### ISSUE THREAD

**Issue by ekids1234** [Plailect/Guide#720]
I had seen FAQ, but I can't solve it. 

(New 3DS LL(JPN) and it has been installed A9LH.)

My friend didn't know that using 64G microSD, he need to change it to FAT32.
(he use 32G before.)
It is inevitable that he can't boot, and he try to use old sdcard to boot, just black screen.
(I also did "Clear Home Menu’s extdata by navigating to the following folder on your SD card" every time.)

We have tried many way,including trying 9.2.0 ctrtransfer, restore the previous backup,or use Luma v5.2's
bin to boot,and we just get black screen.

Some "feature" with this 3DS : 
1. I can enter Luma setting with holding select, entering H9 with holding start. 
(Is this point out that this 3DS is not brick?)
2. When it is booting, we can heard the boot sound. 
3. After booting, even if I can't see anything ,I can just enter game (CTR). 
how did i know ? Because I insert my pokemon CTR, repeat to click A and I heard the game sound... 
4. If I hold L+R+UP+A, I can enter Update interface. It is so strange !!!! 
   Why can't it show the home menu but show update interface ? 
I just choose exit, because I am so afraid that will overwrite something.
(now, in theory, I am at 9.2.0-20J , because I did 9.2.0 ctrtransfer.)

this is operation video : 
https://www.youtube.com/watch?v=mwPV4Sk5ucU
[[mirror](https://drive.google.com/file/d/0BxVKCTzqN-JIX3pibU9IX3hpMlE/view?usp=sharing)]

Sorry for my poor English, please help me to solve it.

**Comment by Plailect** [comment]
This is an extremely old version of Luma3DS.

Do https://plailect.github.io/Guide/updating-a9lh

**Comment by ekids1234** [comment]
The answer is "going to update",but I am in the latest version.
To check version can watch [this video](https://drive.google.com/open?id=0BxVKCTzqN-JIWk5CaHFYQTltTHM).
(https://drive.google.com/open?id=0BxVKCTzqN-JIWk5CaHFYQTltTHM)

I also tried to follow FAQ-and-troubleshooting in Luma3DS wiki,but even if I toggle the Developer options to ErrDisp, it doesn't show anything.
(Wiki say "If you see a "svcBreak" in the "loader" process, go ahead with the steps",but I just get black screen.)

If I did wrong steps ? 
Or in that video, I am not in least version ?
